cmd/benchmark/main.go

```diff
@@ -401,7 +401,7 @@ func main() {
 		totalBalanceAfterWithFee := big.NewInt(0)
 		for nretries := 0; nretries < 3; nretries++ {
 			totalBalanceAfter, _ = getTotalBalance(insSDK, members)
-			totalBalanceAfterWithFee = new(big.Int).Add(totalBalanceAfter, big.NewInt(calcFee(transferAmount)))
+			totalBalanceAfterWithFee = new(big.Int).Add(totalBalanceAfter, big.NewInt(calcFee(transferAmount)*int64(repetitions)))
```

Comment:
Repetitions is not equal to count of transfer operations. Repetitions - count of transfer for one user (as it written in flag description). Count of transfer operations will be `repetitions*concurrent`.

Reply:
Thank you! fixed